tests/testthat/test-crunch-data-frame-dfconformance.R

```diff
@@ -0,0 +1,45 @@
+context("CrunchDataFrame conformance with data.frame methods")
```

Comment:
I don't particularly care for this file name. And I'm not sure the value of moving these couple of tests out of test-as-data-frame.R